prow/tide/tide.go

```diff
@@ -761,7 +718,7 @@ func filterPR(ghc githubClient, mergeAllowed func(*CodeReviewCommon) (string, er
 // the requested method and does not have a merge conflict.
 // It caches results and should be cleared periodically with clearCache()
 //
-// This struct is GitHub specific
+// This struct is GitHub specific, and should be used only by GitHubProvider.
```

Comment:
Please collocate this with the GitHubProvider then.

Reply:
will do once this PR is approved